static/scss/partials/glocal-menu.scss

```diff
@@ -17,16 +17,16 @@ glocal-menu {
   position: absolute;
   visibility: hidden;
   color: #000000;
-  right: 28px;
+  right: $spacing-lg;
   border-radius: 8px;
   box-shadow: 0 7px 12px -3px rgba(28, 28, 29, .502);
   width: auto;
-  max-width: $screen-sm;
+  max-width: calc(#{$screen-sm} - #{$spacing-lg});
 
-    @media screen and (max-width: 520px) {
-      max-width: 100%;
-      left: 16px;
-      right: 16px;
+    @media screen and (max-width: $screen-sm) {
+      max-width: calc(#{$screen-sm} - #{$spacing-md} - #{$spacing-md});
```

Comment:
This line fine-tunes the mobile view, I switched the left/right padding to a smaller `$spacing-md` (16px) value and accounted for this padding in the mobile `max-width`. Preview: https://drive.google.com/file/d/1Q-EPRZMdOfrIFSQXbC7YKEFFy_FIS4gu/view?usp=sharing